v1.0/v1.0/exit-success.cwl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+inputs: []
+baseCommand: /bin/false
```

Comment:
can we change it to just `"false"`? it is /usr/bin/false on MacOS

Reply:
Sure